test/controllers/import.test.js

```diff
@@ -390,4 +392,59 @@ describe('ImportController tests', () => {
       expect(response.headers.get('x-error')).to.equal('Presigner error');
     });
   });
+
+  describe('getImportJobsByDateRange', () => {
+    it('should throw an error when no jobs are found', async () => {
+      requestContext.data.startDate = '2022-10-05T14:48:00.000Z';
+      requestContext.data.endDate = '2022-10-07T14:48:00.000Z';
+      const response = await importController.getImportJobsByDateRange(requestContext);
+      expect(response).to.be.an.instanceOf(Response);
+      expect(response.status).to.equal(404);
+      expect(response.headers.get('x-error')).to.equal('Not found');
```

Comment:
Per the above comments, I wonder if this response should be an HTTP 200 with an empty array instead.

Reply:
agreed